### PR TITLE
Update mam-and-mdm.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed the hyper link

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9931

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

File: microsoft-365/business/ui/mam-and-mdm.md [hunk 1]
```diff
@@ -28,13 +28,13 @@ description: "Learn the differences between mobile device management and mobile
...
 
-Microsoft 365 Business Premium offers a number of ways for you to protect your business data. See [Overview of Microsoft 365 Business Premium](../../admin/admin-overview/what-is-microsoft-365.md) for more about the various protections that are automatically set up, and what you can set up yourself to further protect your business. You can also set up policies that protect your Windows 10 devices and the data in your mobile devices.
-[Set application protection settings for Windows 10 devices](../../business-premium/m365bp-protection-settings-for-windows-10-devices.md).
+Microsoft 365 Business Premium offers a number of ways for you to protect your business data. See [Overview of Microsoft 365 Business Premium](../../admin/admin-overview/what-is-microsoft-365.md) for more about the various protections that are automatically set up, and what you can set up yourself to further protect your business. You can also onboard devices to Microsoft Defender for Business to protect your Windows devices and the data in your mobile devices. See 
+[Onboard enrolled devices to Microsoft Defender for Business](/microsoft-365/business-premium/m365bp-onboard-devices-mdb).
```
Comment (tiburd) [inline microsoft-365/business/ui/mam-and-mdm.md]:
```suggestion
```

File: microsoft-365/business/ui/mam-and-mdm.md [hunk 2]
```diff
@@ -28,13 +28,13 @@ description: "Learn the differences between mobile device management and mobile
 
 # Difference between MDM and MAM
 
-Microsoft 365 Business Premium offers a number of ways for you to protect your business data. See [Overview of Microsoft 365 Business Premium](../../admin/admin-overview/what-is-microsoft-365.md) for more about the various protections that are automatically set up, and what you can set up yourself to further protect your business. You can also set up policies that protect your Windows 10 devices and the data in your mobile devices.
-[Set application protection settings for Windows 10 devices](../../business-premium/m365bp-protection-settings-for-windows-10-devices.md).
+Microsoft 365 Business Premium offers a number of ways for you to protect your business data. See [Overview of Microsoft 365 Business Premium](../../admin/admin-overview/what-is-microsoft-365.md) for more about the various protections that are automatically set up, and what you can set up yourself to further protect your business. You can also onboard devices to Microsoft Defender for Business to protect your Windows devices and the data in your mobile devices. See
```
Comment (tiburd) [inline microsoft-365/business/ui/mam-and-mdm.md]:
```suggestion
Microsoft 365 Business Premium offers a number of ways for you to protect your business data. See [Overview of Microsoft 365 Business Premium](../../admin/admin-overview/what-is-microsoft-365.md) for more about the various protections that are automatically set up, and what you can set up yourself to further protect your business. You can also onboard devices to Microsoft Defender for Business to protect your Windows devices and the data in your mobile devices. For more information, see [Onboard enrolled devices to Microsoft Defender for Business](/microsoft-365/business-premium/m365bp-onboard-devices-mdb). 
```

**Pull request merged**